.cargo/config.toml

```diff
@@ -5,7 +5,7 @@
 # process is deterministic.
 #
 # See https://linux.die.net/man/1/ld (--build-id flag).
-rustflags = ["-C", "link-args=-Xlinker --build-id=none"]
+rustflags = ["-C", "link-args=-Xlinker --build-id=none", "--cfg", "tokio_unstable"]
```

Comment:
As often, I think your useful details from the commit message would be even more useful as comments in this file.

Reply:
I'll try to not add as much information into commit messages in the future. 🙃

Reply:
TBH I think your commit messages are often too valuable to get buried into the git history -- I would like them to be seen more easily by others and future maintainers.